### DISCUSSION TITLE
Add support for firefox

### DISCUSSION THREAD

**Discussion by FluffyDiscord** [spatie/browsershot#712] [Ideas]
Right now this package is hardcoded to use chromium. Option to use firefox would be nice, as it actually handles big pages/html tables, instead of crashing like chrome does.

**Comment by georgyKurian** [comment]
Chrome doesn't work on Linux with arm64, which is there on AWS. 
https://support.google.com/chrome/a/answer/7100626?hl=en

**Comment by lucidlemon** [comment]
This sadly still seems to happen.

Very unlucky since chromium handles table headers repeating on serveral pages poorly. And since the headerView and headerHtml crash when you try to use css it pretty much makes repeating headers VERY difficult.
